identity-apps-core/apps/authentication-portal/src/main/webapp/includes/username-label-resolver.jsp

```diff
@@ -49,6 +57,8 @@
                     i18nKey = "email";
                 } else if (StringUtils.equals(attribute, MOBILE_CLAIM_URI)) {
                     i18nKey = "mobile";
+                } else {
+                    i18nKey = getClaimDisplayName(attribute, tenantDomain);
```

Comment:
Can we fetch all claims in a single call and then rely on a map afterward?
But let us discuss this further, because with this approach we might end up making an unnecessary call in most common cases like mobile / username / email